tensorflow/contrib/sparsemax/python/ops/sparsemax.py

```diff
@@ -85,7 +85,7 @@ def sparsemax(logits, name=None):
     p = math_ops.maximum(
         math_ops.cast(0, logits.dtype), z - tau_z[:, array_ops.newaxis])
     # If k_z = 0 or if z = nan, then the input is invalid
-    p_safe = array_ops.where(
+    p_safe = array_ops.where_v2(
```

Comment:
The tests for this are failing because they rely on the old broadcasting behavior of tf.where. Please fix.

Reply:
@alextp thanks for the comments, do you recommend to revert back the change ?

Regards
Amit